packages/samples/react/src/components/SampleDescription.tsx

```diff
@@ -1,23 +1,27 @@
+import './SampleDescription.css';
 import type { FC, PropsWithChildren } from 'react';
-import React, { useContext } from 'react';
+import React, { useContext, useMemo } from 'react';
+
+import { PUBLIC_DOC_COMPONENT_URL } from '../shares/constants';
 
 import { KolIndentedText, KolLink } from '@public-ui/react';
 
 import { HideMenusContext } from '../shares/HideMenusContext';
 
 export const SampleDescription: FC<PropsWithChildren> = (props) => {
 	const hideMenus = useContext(HideMenusContext);
+	const docRedirect = useMemo(() => {
+		const arr = location.href.split('/');
+		return `${PUBLIC_DOC_COMPONENT_URL}/${arr[arr.length - 2]}`;
+	}, [PUBLIC_DOC_COMPONENT_URL, location.href]);
 
 	return hideMenus ? null : (
 		<div className="flex mb-sm">
 			<KolIndentedText>{props.children}</KolIndentedText>
-			<KolLink
-				_hideLabel
-				_href={`${location.href}?hideMenus`}
-				_label="Beispiel in neuem Tab öffnen"
-				_target="_blank"
-				className="mla flex-self-center"
-			></KolLink>
+			<div className="links">
+				<KolLink _hideLabel _href={docRedirect} _label="Dokumentation in neuem Tab öffnen" _target="_blank" className="mla flex-self-center" />
+				<KolLink _hideLabel _href={`${location.href}?hideMenus`} _label="Beispiel in neuem Tab öffnen" _target="_blank" className="mla flex-self-center" />
```

Comment:
Ich empfände es als besser, wenn das Label nicht versteckt wird und dafür kürzer: „Dokumentation“ und „Beispiel“. Dass es in einem neuen Tab geöffnet wird, zeigt das Icon an.